source/velocity_boundary_conditions/gplates.cc

```diff
@@ -191,21 +177,13 @@ namespace aspect
         AssertThrow(dn_theta - n_theta <= 1e-5,
                     ExcMessage("The velocity file has a grid structure that is not readable. Please refer to the manual for a proper grid structure."));
 
-        if ((delta_theta != 0.0) || (delta_phi != 0.0))
-          {
-            Assert((delta_theta - numbers::PI / (n_theta-1)) <= 1e-7,  ExcMessage("Resolution changed during boundary conditions. Interpolation is not working correctly."));
-            Assert((delta_phi - 2*numbers::PI / n_phi) <= 1e-7,  ExcMessage("Resolution changed during boundary conditions. Interpolation is not working correctly."));
-          }
-
         delta_theta =   numbers::PI / (n_theta-1);
         delta_phi   = 2*numbers::PI / n_phi;
 
-        // swap pointers to old and new values, we overwrite the old ones
-        // and the new ones become the old ones
-        std::swap (velocity_values, old_velocity_values);
-
-        (*velocity_values).reinit(n_theta,n_phi);
-        velocity_positions.reinit(n_theta,n_phi);
+        /**
+         * Tables which contain the velocities
```

Comment:
Can you clarify in this comment, that is holds the velocities at every point, and that we have two tables, one for the theta component and one for the phi component? By the way, which one is which? I do not remember, we should have documented that right away ;-)